news/PysideLegacyIssues.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* <news item>
+
+**Deprecated:**
+
+* Updated a few deprecated instances of app.exec()
+
+**Removed:**
+
+* <news item>
+
+**Fixed:**
+
+* Fixed several bugs resulting from transfer from Pyside2 to Pyside6
```

Comment:
This is good.  The message is for users and devs to know hat was changed in the new release, so I would probably simply write here something like `"Upgraded from PySide2 to PySide6"`